src/databricksbundle/detector.py

```diff
@@ -9,3 +11,25 @@ def is_databricks():
     :return: true, if running on databricks
     """
     return os.getenv("DATABRICKS_RUNTIME_VERSION") is not None
+
+
+def is_jupyter_server_running():
+    """
+    Check, if the code is executed by the Jupyter lab instance or locally.
+    :return: true, if running on Jupyter server
+    """
+    return "ipykernel" in sys.modules
+
+
+def jupyter_cwd_setter():
+    """
+    List currently running Jupyter servers and set the current working directory to the root directory of project running the notebook.
+    :return: updated cwd to project root
+    """
+    if is_jupyter_server_running():
+        from jupyter_server import serverapp
+
+        running_servers = list(serverapp.list_running_servers())
+        for item in running_servers:
+            if Path(item["root_dir"]).resolve() in Path.cwd().parents:
```

Comment:
Does this really work in a way that the best matching path is used? I wouldn't say so...